dev-app-update.yml

```diff
@@ -0,0 +1,3 @@
+owner: adlk
+repo: simplenote-test
```

Comment:
Are you adding this file temporarily for a build test?

Reply:
Oh, good catch! This file should actually be ignored in .gitignore but this isn't merged yet 🤦‍♂️